app/models/katello/content_view.rb

```diff
@@ -316,6 +317,14 @@ def duplicate_puppet_modules
       counts.select { |_k, v| v > 1 }.keys
     end
 
+    def check_non_composite_components
+      unless composite?
+        unless components.blank?
```

Comment:
This seems like it would prevent me from adding components to a composite view that already has some components added to it?